minikube/minikube.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash 
+set -x
+set -eu
+set -o pipefail
+
+up_registry () {
+  docker run -d -p 5000:5000 --restart=always --name registry registry:2
+}
+
+deploy_es () {
+
+  if [[ $(kubectl get namespace | grep oss | wc -l) -lt 1 ]]; then
+    kubectl create namespace oss
+  fi
+
+  mkdir -p elasticsearch
+  pushd elasticsearch
+  for YAML in service-account.yaml es-svc.yaml es-rc.yaml ; do
```

Comment:
You leave my comment unnoticed about keeping these files locally and do not try to download them.